docs/Docker_image.md

```diff
@@ -0,0 +1,24 @@
+# Elección de la imagen de docker.
+
+¿Qué se busca en una imagen?
+
+- Estabilidad.
+- Tamaño de la imagen.
+- Actualizado al día con el lenguaje, en este caso python.
+- Actualizada las dependencias.
+
+Partimos de dos caminos:
+
+- **Oficiales de docker**
+
+    - Alpine. Útil para imágenes pequeñas, pero puede conllevar a errores de rendimiento y tamaño por la gestión de paquetes.
+    - Slim. Carece de las capas en los paquetes comunes, por lo que hace que sea una imagen mucho más pequeña.
+    - Bullseye. Tiene muchos paquetes instalados por capas de imagen que usan otras imágenes de docker, lo que hace que el uso del disco sea bajo.
+
+- **Otras**
+
+    - Ubuntu. Imagen muy pesada en comparación con las anteriores y tiene muchos paquetes instalados que no son necesarios para este proyecto.
+    - Debian. Más liviana que la anterior, pero para usar Debian es más conveniente usar una versión de las oficiales de python que usan Debian de manera más liviana, como el caso de la versión bullseye.
+    - **bitnami** Esta será la elegida. Además de que es una imagen más ligera que las demás mencionadas, buen rendimiento, muchas descargas en dockerhub, cuenta además con el 'VERIFIED PUBLISHER' de dockerhub en python. Otro motivo ajeno es diferenciarse de las oficiales de python que serán elegidas por otros compañeros...
```

Comment:
¿y la estabilidad? Si ponéis algo usadlo (o no pongáis cosas que no usáis)

Reply:
:arrow_up:

Reply:
En este casi lo has quitado, así que resuelve el comentario poniendo el commit en el que lo has cambiado.

Reply:
Está hecho el [commit](https://github.com/MauronMP/PMP_IV/pull/36/commits/a8148657dd85e213653a7b03b857d3cb48d64d0b) dónde elimino la línea y lo asigno a un issue